src/main/java/org/folio/services/item/ItemService.java

```diff
@@ -176,4 +185,27 @@ public ItemService(Context vertxContext, Map<String, String> okapiHeaders) {
   private Function<SQLConnection, Future<RowSet<Row>>> updateSingleItemBatchFactory(Item item) {
     return connection -> itemRepository.update(connection, item.getId(), item);
   }
+
+  private Future<Item> refuseWhenStatisticalCodeIdDoesNotExist(final Item item) {
```

Comment:
Can you add a comment to this pull request's description whether there will be an automated migration for non-existing statistical code ids? Or state that a warning will be added to the release notes that SysOps need to manually search for non-existing statistical code ids and replace them by an existing one, otherwise updating an item will be rejected?

Reply:
I think this is a question for the product owner. I would suggest a migration task to remove the non existent ids in use.

Reply:
Moved to issue.